logging.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_iam_role_policy" "instance" {
-  count  = var.enable_cloudwatch_logging ? 1 : 0
-  name   = "${local.name_iam_objects}-instance-role"
+  count  = var.enable_cloudwatch_logging && var.create_runner_iam_role ? 1 : 0
```

Comment:
When using a custom instance role, it is useful to have "enable cloudwatch logging=true" so that the instance will produce logs, but not attach the IAM permissions since the role and policies are managed outside of the module.